images/federation/Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# This is the OpenShift Origin Federation image, used for running the
+# federation apiserver and controller manager components.
+#
+# The standard name for this image is openshift/origin-federation
+#
+FROM openshift/origin-base
```

Comment:
If you expect the `hyperkube` binary to be shipped to customers it will be in the RPM so install using `yum` _a la_ `origin-pod`.